### PR TITLE
Replace vtctld2 screenshots and delete removed flags

### PR DESCRIPTION
**Pull request by notfelineit**
This PR replaces screenshots of vtctld2 UI with screenshots from the VTAdmin UI. It also updates vtctld UI references in our docs to reference VTAdmin instead, and removes deleted flags from the example code blocks.

### CODE REVIEW

File: content/en/docs/16.0/reference/features/topology-service.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ path=/keyspaces/ks2/Keyspace version=55
 sharding_column_name: "col2"
 ```
 
-The `vtctld` web tool also contains a topology browser (use the `Topology`
+The VTAdmin web tool also contains a topology browser (use the `Topology`
 tab on the left side). It will display the various proto files, decoded.
```
Comment (deepthi) [inline content/en/docs/16.0/reference/features/topology-service.md:228]: Is the last sentence still accurate?

File: content/en/docs/16.0/reference/features/transport-security-model.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Vitess exposes a few RPC services and internally uses RPCs. These RPCs can optio
 
 The following diagram represents all the RPCs we use in a Vitess cluster via gRPC:
 
-![Vitess Transport Security Model Diagram](../../img/vitesstransportsecuritymodel.svg)
+![Vitess Transport Security Model Diagram](../../img/vitesstransportsecuritymodel.png)
```
Comment (deepthi) [inline content/en/docs/16.0/reference/features/transport-security-model.md:13]: I wonder how accurate this diagram is now.
Maybe replace vtctlclient with vtctldclient?
@rohit-nayak-ps @mattlord is `binlogstreamer` still accurate?

File: content/en/docs/16.0/user-guides/configuration-basic/create-cell.md [hunk 3]
```diff
@@ -45,14 +45,12 @@ If you have deployed in multiple regions and would like the flexibility of queri
 ## Checklist
 
 * Ensure that vtctlds come up successfully. If there is a failure, check the log files for any errors.
-* Ensure that you can browse to the http port of vtctld. The dashboard should appear, after which you should be able to browse under the `Topology` tab and verify that the cell information is as you created it.
+* Ensure that you can query the http port of vtctld: `curl http://localhost:15000/cells/`
 * If you configured a separate cell-specific topo, ensure that you can connect to it using the parameters in the cell information.
 * Ensure that the cell-specific topos are reachable from other cells.
 
+If you have VTAdmin running, open the VTAdmin application in your browser. Go to Topology > View Topology
```
Comment (ajm188) [inline content/en/docs/16.0/user-guides/configuration-basic/create-cell.md]: IMO, we should write this in a way that doesn't make it sound like vtadmin is optional, so just: "Open the VTAdmin application in your browser. Go to ..."

File: content/en/docs/16.0/user-guides/configuration-basic/ports.md [hunk 4]
```diff
@@ -29,6 +29,8 @@ for the various components:
       * TCP port 16000 + vttablet UID (gRPC); e.g port 16100 for UID 100
     * vtctldclient &rarr; vtctld
       * TCP port 15999 (gRPC)
+    * vtadmin &rarr;
```
Comment (ajm188) [inline content/en/docs/16.0/user-guides/configuration-basic/ports.md:32]: rawwwrrrr

File: content/en/docs/16.0/user-guides/configuration-basic/reparenting.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ Once you have the cluster up and running, you should perform a trial failover us
 
 A typical use case for `PlannedReparentShard` is to use it during software updates. The command has a convenient `avoid_tablet` flag that allows you to specify the current vttablet you are going to perform maintenance on. If that is a primary, then it performs a failover to another eligible replica. Otherwise, it is a no-op.
 
-You can also perform a "Planned Reparent" through the browser from the `vtctld` Dashboard.
+You can also perform a "Planned Reparent" through the browser from the VTAdmin Dashboard.
 
-![vtctld-reparenting](../img/vtctld-reparenting.png)
+![vtadmin-reparenting](../img/vtadmin-reparenting.png)
```
Comment (deepthi) [inline content/en/docs/16.0/user-guides/configuration-basic/reparenting.md:12]: This screenshot doesn't actually show `Planned Reparent`.

**Review by ajm188 — APPROVED** [review]
LGTM, just one minor suggestion!

**Review by deepthi — APPROVED** [review]
Can be merged once all feedback has been addressed.

File: content/en/docs/16.0/user-guides/configuration-basic/create-cell.md (hunk 3 above)
Comment (notfelineit) [reply content/en/docs/16.0/user-guides/configuration-basic/create-cell.md]: That's a good idea!

File: content/en/docs/16.0/user-guides/configuration-basic/reparenting.md (hunk 5 above)
Comment (notfelineit) [reply content/en/docs/16.0/user-guides/configuration-basic/reparenting.md:12]: I'll update it!

File: content/en/docs/16.0/reference/features/topology-service.md (hunk 1 above)
Comment (notfelineit) [reply content/en/docs/16.0/reference/features/topology-service.md:228]: Screen Shot 2022-12-09 at 12 30 49 PM
Are these the decoded files it meant? If not, I can take that last sentence out

File: content/en/docs/16.0/user-guides/configuration-basic/ports.md (hunk 4 above)
Comment (notfelineit) [reply content/en/docs/16.0/user-guides/configuration-basic/ports.md:32]: 🐱

File: content/en/docs/16.0/reference/features/topology-service.md (hunk 1 above)
Comment (notfelineit) [reply content/en/docs/16.0/reference/features/topology-service.md:228]: The underlying logic for fetching this information hasn't changed much from the way it was fetched for the vtctld2 UI: https://github.com/vitessio/vitess/blob/main/go/vt/vtctl/grpcvtctldserver/server.go#L4365

File: content/en/docs/16.0/reference/features/transport-security-model.md (hunk 2 above)
Comment (mattlord) [reply content/en/docs/16.0/reference/features/transport-security-model.md:13]: LGTM. The related RPCs support TLS:
```
      --binlog_player_grpc_cert string                                   the cert to use to connect
      --binlog_ssl_cert string                                           PITR restore parameter: Filename containing mTLS client certificate to present to binlog server as authentication.
```

I think that's where we're emulating [MySQL's replication with SSL](https://dev.mysql.com/doc/refman/8.0/en/replication-encrypted-connections.html) support.

**Pull request merged**